x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/per_field_rule_diff_tab.tsx

```diff
@@ -43,16 +43,32 @@ export const PerFieldRuleDiffTab = ({ ruleDiff }: PerFieldRuleDiffTabProps) => {
     <>
       <RuleDiffHeaderBar />
       {aboutFields.length !== 0 && (
-        <RuleDiffSection title={i18n.ABOUT_SECTION_LABEL} fieldGroups={aboutFields} />
+        <RuleDiffSection
+          title={i18n.ABOUT_SECTION_LABEL}
+          fieldGroups={aboutFields}
+          dataTestSubj="perFieldDiffAboutSection"
+        />
       )}
       {definitionFields.length !== 0 && (
-        <RuleDiffSection title={i18n.DEFINITION_SECTION_LABEL} fieldGroups={definitionFields} />
+        <RuleDiffSection
+          title={i18n.DEFINITION_SECTION_LABEL}
+          fieldGroups={definitionFields}
+          dataTestSubj="perFieldDiffDefinitiontSection"
```

Comment:
There is a misprint here `Definitiont`, replace `perFieldDiffDefinitiontSection` with `perFieldDiffDefinitionSection`.